redirects.yaml

```diff
@@ -533,6 +533,13 @@ minigamemadnesswiki:
   ca: 'Sectigo'
   hsts: 'strict'
   disable_event: true
+gimkitwiki:
```

Comment:
This unfortunately conflicts with the entry in the certs.yaml file, therefore we cannot use this as the key name, maybe gimkitredirect?